Packs/ForescoutEyeInspect/IncidentFields/incidentfield-ForescoutEyeInspect_Destination_MAC_Address.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
@RazOmry We are trying to minimize the creation of new fields. Please use the common field "Destination MAC Address" instead.